### DISCUSSION TITLE
I got a problem when i output the httpx result

### DISCUSSION THREAD

**Discussion by 8ramhaS3C** [projectdiscovery/httpx#1875] [Q&A]
when i output the result and view with nvim . the result is not in palin format. some ansii character like [3m][[44m] etc got appended in result.

**Comment by GeorginaReeder** [comment]
Hi @8ramhaS3C ! Thanks for your question.

As a starting point, what's your input? If you can let us know that, along with any screenshots you have of what the output looks like exactly, that would be really helpful and we can take a closer look from there. Thanks!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
